editor-components/Image_gallery/image_gallery_5/image-gallery5.tsx

```diff
@@ -0,0 +1,131 @@
+import * as React from "react";
+import { BaseImageGallery } from "../../EditorComponent";
+import styles from "./image-gallery5.module.scss";
+
+interface ImageType {
+  type: string;
+  key: string;
+  displayer: string;
+  value: string;
+}
```

Comment:
Do not use interface, use type. `type ImageType = {`

[Read this file](https://github.com/Teknodev/composer-tools/blob/main/editor-components/README.md)